src/components/CloseVaultDialog.tsx

```diff
@@ -130,7 +120,7 @@ const CloseVaultDialog = ({
           {...noticeProps}
           className="overflow-hidden text-interOrange"
         >
-          Offer sent to Agoric Smart Wallet for approval.
+          Vault has been closed successfully.
```

Comment:
It's only been sent, right? The transaction could still fail. For example if another transaction in flight took out more debt, this pay down to close would not be enough.

Reply:
Oh just the variable name is off. Approving assuming you'll revise

Reply:
Updated the variable name